logging/src/main/scala/com/ovoenergy/effect/Logging.scala

```diff
@@ -31,17 +31,23 @@ object Logging {
     def tags: Tags
   }
 
-  case class Debug(of: String, tags: Tags = Map.empty) extends Log
-  case class Info(of: String, tags: Tags  = Map.empty) extends Log
-  case class Error(of: String, tags: Tags = Map.empty) extends Log
+  case class Debug(of: String, tags: Tags                 = Map.empty) extends Log
+  case class Info(of: String, tags: Tags                  = Map.empty) extends Log
+  case class Error(of: Ior[String, Throwable], tags: Tags = Map.empty) extends Log
```

Comment:
suspicious indentation anyway 🤔

Reply:
Seems to have been indented automatically by scalafmt based on the rules we have defined in the project. 🤔

Reply:
it'd be nice to fix whatever rule is causing that 😨

Reply:
Done in d9004d8.